tests/e2e/multi_svc_test.go

```diff
@@ -0,0 +1,658 @@
+		}()
+
+		ginkgo.By("Verify datastore has permission for storage service account from supervisor cluster")
+		userPermission, err := verifyPermissionForWcpStorageUser(dataCenter, datastoreName, "", "", wcpServiceAccUsers[0], roleCnsDatastore)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(userPermission).To(gomega.BeTrue(), "user permission is not changed for cluster")
+
+		ginkgo.By("Unmount datastore from one of the host from supervisor cluster")
+		clusterComputeResource, _, err := getClusterName(ctx, &e2eVSphere)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		hostsInCluster := getHostsByClusterName(ctx, clusterComputeResource, computeCluster)
+		hostIP1, err := hostsInCluster[0].ManagementIPs(ctx)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		hostToBeRemoved := hostIP1[0].String()
+		hostPath := computeClusterPaths[0] + "/" + hostToBeRemoved
+		framework.Logf("Unmount datastore from host : %v", hostToBeRemoved)
+		err = UnMountDatastoreFromClusterOrHost(datastoreName, hostToBeRemoved)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		unMount = true
+		defer func() {
+			if unMount {
+				mountDatastoreOnClusterOrHost(datastoreName, datastoreIP, hostPath)
+			}
+		}()
+
+		ginkgo.By("Verify an alarm is raised for unmounted datastore and host in the clsuter")
+		alarm := "Datastore not accessible to all hosts under the cluster"
+		err = verifyAlarmFromDatacenter(dataCenter, alarm)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Remove host from the cluster from STEP 3 and Verify alarm has disappeared")
+		enterHostIntoMM(ctx, hostsInCluster[0], ensureAccessibilityMModeType, timeout, false)
+		hostInMM = true
+		defer func() {
+			if hostInMM {
+				exitHostMM(ctx, hostsInCluster[0], timeout)
+			}
+		}()
+
+		isHostRemoved, err := removeEsxiHostFromCluster(dataCenter, computeCluster, hostToBeRemoved)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		gomega.Expect(isHostRemoved).To(gomega.BeTrue(), "Host was not removed from cluster")
+		defer func() {
+			if isHostRemoved {
+				err = moveHostToCluster(computeClusterPaths[0], hostToBeRemoved)
+				gomega.Expect(err).NotTo(gomega.HaveOccurred())
+			}
+		}()
+
+		err = verifyAlarmFromDatacenter(dataCenter, alarm)
+		gomega.Expect(err).To(gomega.HaveOccurred())
+
+		ginkgo.By("Add host back to cluster and Verify alarm has appeared again")
+		err = moveHostToCluster(computeClusterPaths[0], hostToBeRemoved)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		isHostRemoved = false
+		exitHostMM(ctx, hostsInCluster[0], timeout)
+		hostInMM = false
+		err = verifyAlarmFromDatacenter(dataCenter, alarm)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+
+		ginkgo.By("Mount datastore back to host in the cluster and Verify alarm has disappeared")
+		err = mountDatastoreOnClusterOrHost(datastoreName, datastoreIP, hostPath)
+		gomega.Expect(err).NotTo(gomega.HaveOccurred())
+		unMount = false
+		time.Sleep(30 * time.Second)
```

Comment:
remove static sleep

Reply:
removed and handled inside method using polling